src/main/java/io/cdap/e2e/utils/WaitHelper.java

```diff
@@ -267,4 +267,20 @@ public static boolean waitForTextToBePresentInElementValue(WebElement element, S
                   " with the Default timeout: " + ConstantsUtil.DEFAULT_TIMEOUT_SECONDS + " seconds");
     return SeleniumDriver.getWaitDriver().until(ExpectedConditions.textToBePresentInElementValue(element, text));
   }
+
+  /**
+   * Wait for the element to be click-disabled
```

Comment:
Wait for the element to be click-disabled to ```Wait for the element to be disabled```

Reply:
Changed to the suggested text